.clang-format

```diff
@@ -41,7 +41,7 @@ IndentCaseBlocks: true
 IndentExternBlock: Indent
 IndentPPDirectives: AfterHash
 IndentWidth: 4
-InsertBraces: true
```

Comment:
Actually, if we could leave this as `true`, I'd appreciate it. This is one of the few that I actually care about. I find it visually difficult when I'm scanning the code if braces are missing around single statements.